script/Ciel/Day 14/14_01A_CIEL14A_6.txt

```diff
@@ -276,62 +299,62 @@ That's it.
 [sha:41b84ad6e73a841dd384178791f52f21c19b8f3e]{
 -- Page 223, Offset 38300.
 -- もう何年も前に落ちたことのある、
-I sink into the deep, deep darkness,
+Just as I did many years ago―――
 }
 [sha:9908e81ca9b0a8b032a2bc69a3e60732eb1d4050]{
 -- Page 223, Offset 38301.
 -- 深い深い闇へと落ちていけた。
-that I once fell into many years ago.
+I sink once more into the deep, deep, darkness.
```

Comment:
Still think reversing it is preferrable
Can also move 'once' to the first clause
>I sink in the deep, deep, darkness―――

>Just as I once did many years ago.